homeassistant/components/pvpc_hourly_pricing/config_flow.py

```diff
@@ -1,22 +1,49 @@
 """Config flow for pvpc_hourly_pricing."""
 from __future__ import annotations
 
+from collections.abc import Mapping
 from typing import Any
 
+from aiopvpc import DEFAULT_POWER_KW, PVPCData
 import voluptuous as vol
 
 from homeassistant import config_entries
+from homeassistant.const import CONF_API_TOKEN, CONF_NAME
 from homeassistant.core import callback
 from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.util import dt as dt_util
 
-from . import CONF_NAME, UI_CONFIG_SCHEMA, VALID_POWER
-from .const import ATTR_POWER, ATTR_POWER_P3, ATTR_TARIFF, DOMAIN
+from .const import (
+    ATTR_POWER,
+    ATTR_POWER_P3,
+    ATTR_TARIFF,
+    CONF_USE_API_TOKEN,
+    DEFAULT_NAME,
+    DEFAULT_TARIFF,
+    DOMAIN,
+    VALID_POWER,
+    VALID_TARIFF,
+)
+
+_MAIL_TO_LINK = (
+    "[consultasios@ree.es]"
+    "(mailto:consultasios@ree.es?subject=Personal%20token%20request)"
```

Comment:
It's copied from the API docs (first link in https://api.esios.ree.es, which includes that specific subject) 🤷‍♂️

(also, it wouldn't sound better if translated to Spanish, believe me 🙈, there is a lot of _spanglish_ integrated in the tech world, and words like 'token' are never translated, or it would be less understandable 😅)